graphql_compiler/schema/sqlalchemy_schema.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019-present Kensho Technologies, LLC.
+from collections import namedtuple
+
+
+# Complete schema information sufficient to compile GraphQL queries to SQLAlchemy
+SQLAlchemyMetadata = namedtuple('SQLAlchemyMetadata', (
+    'schema',  # GraphQLSchema
+    'tables',  # dict mapping every graphql type in the schema to a sqlalchemy table
+    'joins',   # dict mapping every graphql type in the schema to:
+               #    dict mapping edge fields at that type to a dict with keys:
```

Comment:
I think that we refer to edge fields as vertex fields in the README.md so let's follow suit here.